Assets/Scripts/GameManagerScript.cs

```diff
@@ -93,60 +108,54 @@ void Awake()
 
 		// Set up arena based on options
 		arenaType = DataManagerScript.arenaType;
-
+	
+		// This is so dumb.
```

Comment:
Haha you could make an array property of all possible arenas, `GameObject[] arenas = new GameObject[8] { Arena1, Arena2, ... };`, and then just do `GameObject selectedArena = arenas[arenaType]; selectedArena.setActive(true);`